src/auro-accordion.js

```diff
@@ -29,7 +29,7 @@ import styleCssFixed from "./style-fixed-css.js";
  * @attr {Boolean} warning - Dependent on shade; warning styling
  * @attr {Boolean} error - Dependent on shade; error styling
  * @attr {Boolean} fixed - Uses px values instead of rem
- * @attr {Boolean} noanimation - Removes the animated opening and closing effect of the accordion
+ * @attr {Boolean} showOverflow - Allow .details content to overflow out of the accordion
```

Comment:
Sorry for, miscommunication on my part. Please add the following line back in

```js
@attr @deprecated {Boolean} noanimation - Removes the animated opening and closing effect of the accordion
```

This removes the line from the documentation, but we still have the feature documented in code and marked for deprecation.

Reply:
Please run `$ npm run apiBuild` to regenerate the API document.